media/js/src/timer.js

```diff
@@ -1,19 +1,28 @@
 let time = $('meta[name="time"]').attr('content');
+let submitted = $('meta[name="submitted"]').attr('content');
 
 function strLength(num) {
     return num.toString().length;
 }
 
 let minutes = Math.floor(time / 60);
 let seconds = Math.floor(time - minutes * 60);
-if (strLength(time) > 2) {
-    $('#quiz-timer').html(
-        `<b>0${minutes}:${seconds}</b> remaining to complete quiz.`);
-
+let display_secs;
+let display_minutes;
+if (strLength(minutes) < 2) {
+    display_minutes = '0' + minutes;
+} else {
+    display_minutes = minutes;
+}
+if (strLength(seconds) < 2) {
+    display_secs = '0' + seconds;
 } else {
-    $('#quiz-timer').html(
-        `<b>${minutes}:${seconds}</b> remaining to complete quiz.`);
+    display_secs = seconds;
 }
+$('#quiz-timer')
+    .html(`<b>${display_minutes}:${display_secs}</b>` +
```

Comment:
Really minor note, but the `<strong>` tag is generally preferred over `<b>` because it's more semantic. Any use of `<b>` can be replaced with `<strong>`.